.gitignore

```diff
@@ -82,7 +82,7 @@ target/
 .python-version
 
 # celery beat schedule file
-celerybeat-schedule
+celerybeat-schedule.db
```

Comment:
Any reason you changed this @solnsumei ?

Reply:
yeah, git did not ignore the `celerybeat-schedule.db` file on my machine until i added the .db extension.

Reply:
Got it @solnsumei . Mine doesn't have a `.db` extension so I'll just include it in my next PR.